CDP4Dal/ISession.cs

```diff
@@ -312,20 +312,13 @@ public interface ISession
         /// </returns>
         Task CloseModelRdl(ModelReferenceDataLibrary modelRdl);
 
-        /// <summary>
-        /// Queries the current <see cref="DomainOfExpertise"/> from the session for the current <see cref="Iteration"/>
-        /// </summary>
-        /// <returns>
-        /// The <see cref="DomainOfExpertise"/> if selected, null otherwise.
-        /// </returns>
-        DomainOfExpertise QueryCurrentDomainOfExpertise();
-
         /// <summary>
         /// Queries the <see cref="Participant"/>'s <see cref="DomainOfExpertise"/>'s from the session for the current <see cref="Iteration"/>

```

Comment:
please update the docs -> current iteration needs to become `provided iteration` and the `returns` statement needs to be updated as well